src/services/copyConfig/consent/consentConfiguration.js

```diff
@@ -117,7 +120,7 @@ class ConsentConfiguration {
   async #copyLegalStatements(destinationSite, destinationSiteConfiguration, consent) {
     const consentId = ConsentConfiguration.#getConsentId(consent.preferences)
     const consentLanguages = consent.preferences[consentId].langs
-    return this.#legalStatement.copy(destinationSite, destinationSiteConfiguration, consentId, consentLanguages)
+    return this.#legalStatement.copy(destinationSite, destinationSiteConfiguration, consentId, consentLanguages, this.#options)
```

Comment:
The class LegalStatement should remain unchanged.
The method #copyConsentStatement can remove the url/link from the consent, according to the option selected, before calling the method this.#copyLegalStatements. Then everything can remain unchanged